cypress/support/fragments/inventory/inventoryMainButton.js

```diff
@@ -1,6 +1,8 @@
-import { Button, Dropdown, Modal } from '../../../../interactors';
+import { Button, Dropdown, Modal, Select, TextArea } from '../../../../interactors';
+import getRandomPostfix from '../../utils/stringTools';
 
 const inventoryApplicationContextDropdown = Dropdown('InventoryApplication context dropdown');
+const instanceTitle = `Instance_Test_Title_${getRandomPostfix()}`;
 
 export default {
```

Comment:
what is inventory main button? think about naming improvement

Reply:
Done